concordium-sdk/src/main/java/com/concordium/sdk/transactions/Payload.java

```diff
@@ -91,7 +91,8 @@ private static UInt64 calculateEnergyCost(int noOfSignatures,
     public enum PayloadType {
         TRANSFER,
         TRANSFER_WITH_MEMO,
-        REGISTER_DATA
+        REGISTER_DATA,
+        INITIALIZE_SMART_CONTRACT_INSTANCE,
```

Comment:
Can we just call it 'INIT_CONTRACT' so it's aligned e.g. with `concordium-base`.

Reply:
Done